frontend/ui/widget/buttondialog.lua

```diff
@@ -80,6 +80,7 @@ local ButtonDialog = InputContainer:extend{
     info_padding = Size.padding.default,
     info_margin = Size.margin.default,
     dismissable = true, -- set to false if any button callback is required
+    flush_events_on_show = false, -- see ConfirmBox for details
```

Comment:
What was the case you met that got you to feel the need to add it here too?

Reply:
I just repeated it from ConfirmBox, following https://github.com/koreader/koreader/pull/7099.
But seems it is not needed, cannot reproduce any issue, so reverted.

Reply:
There was this comment in the section you removed when migrating from a ConfirmBox to ButtonDialog in screenshoter.lua:

https://github.com/koreader/koreader/blob/0d2f8fbb3fc62a8542f483831867e9ba19950d5c/frontend/ui/widget/screenshoter.lua#L45

Have you tried 2 fingers tap ? May be you're better than me at it :) (I haven't tried it recently.) I think it's when doing screenshots, not with a long diagonal swipe, but with taps in diagonaly opposite corners (which has been working, may be it's not a  really documented feature).

Reply:
Yes, 2 fingers tap works good.